pkg/import/helpers.go

```diff
@@ -0,0 +1,44 @@
+package _import
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/pkg/errors"
+)
+
+type CredHelper struct {
+	Auths map[string]authn.Basic
+}
+
+func NewCredHelperFromEnvVars(serverURLEnvVar string, usernameEnvVar string, passwordEnvVar string) *CredHelper {
+	suffixes := [11]string{""}
+	for i := 0; i <= 9; i++ {
+		suffixes[i+1] = "_" + strconv.Itoa(i)
+	}
+
+	auths := map[string]authn.Basic{}
+
+	for _, suffix := range suffixes {
+		if serverUrl := os.Getenv(serverURLEnvVar + suffix); serverUrl != "" {
+			auths[serverUrl] = authn.Basic{
+				Username: os.Getenv(usernameEnvVar + suffix),
+				Password: os.Getenv(passwordEnvVar + suffix),
+			}
+		}
+	}
+
+	return &CredHelper{
+		Auths: auths,
+	}
+}
```

Comment:
To avoid limiting the numbers that we support, could we do something like this? 

```suggestion
	func NewCredHelperFromEnvVars(serverURLEnvVar string, usernameEnvVar string, passwordEnvVar string) *CredHelper {
	auths := map[string]authn.Basic{}

	var re = regexp.MustCompile(fmt.Sprintf(`(%s(_\d+)?)=(.*)`, serverURLEnvVar))
	for _, env := range os.Environ() {
		if match := re.FindStringSubmatch(env); len(match) > 0 {
			auths[match[3]] = authn.Basic{
				Username: os.Getenv(usernameEnvVar + match[2]),
				Password: os.Getenv(passwordEnvVar + match[2]),
			}
		}
	}

	return &CredHelper{
		Auths: auths,
	}
}
```

Reply:
Addressed in commit https://github.com/vmware-tanzu/kpack-cli/pull/279/commits/36fc57bcc87b7a0b021ac4f14be32967c2772f77